### DISCUSSION TITLE
How to set SAP Logon Group in connection params?

### DISCUSSION THREAD

**Discussion by kambalin** [dbosoft/YaNco#320] [Q&A] [answered]
Hello! In old sapnco lib there was "LogonGroup" value in sap connection data. But how to specify this value in yanco? Just write "LogonGroup" or "Group" or somehow else?

**Comment by fw2568** [comment, marked as answer]
tldr: GROUP

YaNco uses the Netweaver RFC library, so all parameters are the same as documented here: https://support.sap.com/content/dam/support/en_us/library/ssp/products/connectors/nwrfcsdk/NW_RFC_750_ProgrammingGuide.pdf 

(Check section RFC client programs)

**Reply by kambalin** [reply]
Thanks, just confirmed my assumption.
